packages/web-components/fast-foundation/docs/integrations/angular.md

```diff
@@ -129,4 +129,10 @@ fast-card > fast-button {
 }
 ```
 
+Third party controls require a ControlValueAccessor for writing a value and listening to changes on input elements. Add ngDefaultControl attribute to your component to have two-way binding working with FormControlDirective, FormControlName or NgModel directives:
```

Comment:
Would welcome @EisenbergEffect’s thought here, but I wonder if this would benefit from the “note” syntax to help it stand out? Thanks for the contribution!

Reply:
Yes, you are right. I have pushed new commit with the "tip" syntax. Happy to help :)

Reply:
After adding the tip syntax the build failed, seems like my syntax is off

Reply:
Changed from "tip" to "note" and build passed